### PR TITLE
Add logging section

### PR DESCRIPTION
**Pull request by dangeross**
This PR splits the `Getting started` section into separate pages and adds an `Adding logging` section

### CODE REVIEW

File: snippets/react-native/getting_started.ts [hunk 1]
```diff
@@ -8,7 +8,9 @@ import {
   mnemonicToSeed,
   type NodeConfig,
   NodeConfigVariant,
-  nodeInfo
+  nodeInfo,
+  LogEntry,
```
Comment (erdemyerebasmaz) [inline snippets/react-native/getting_started.ts:12]: Re: CI error
```
Import "LogEntry" is only used as types
```

```suggestion
  type LogEntry,
```

File: snippets/csharp/GettingStarted.cs [hunk 2]
```diff
@@ -76,4 +76,27 @@ public void FetchNodeInfo(BlockingBreezServices sdk)
...
+        {
+            // Handle error
+        }
+    }
+
+    class SdkLogStream : LogStream
+    {
+        public void Log(LogEntry l)
+        {
+            Console.WriteLine($"Received Log [{l.Level}]: {l.Line}");
```
Comment (erdemyerebasmaz) [inline snippets/csharp/GettingStarted.cs:97]: Re: CI errors
```
'LogEntry' does not contain a definition for 'Level'...
'LogEntry' does not contain a definition for 'Line'...
```

Is this supposed to be:
```suggestion
            Console.WriteLine($"Received Log [{l.level}]: {l.line}");
```

File: src/guide/connecting.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+```cs,ignore
+{{#include ../../snippets/csharp/GettingStarted.cs:init-sdk}}
+```
+</section>
+</custom-tabs>
+
+<div class="warning">
+<h4>Developer note</h4>
+
+By default the config working directory is set to `./`. Some platforms may require that you use an application specific directory that is writable within the application sandbox. For example applications running on Android or iOS.
```
Comment (erdemyerebasmaz) [inline src/guide/connecting.md:82]:
```suggestion
By default the config working directory is set to `./`. Some platforms may require that you use an application specific directory that is writable within the application sandbox. For example, applications running on Android or iOS.
```

File: src/guide/logging.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+# Adding logging
+
+The Breez SDK implements detailed logging via a streaming interface so you can manage within your application. The log entries are split into several levels that you can filter and store how you want within your application, for example appending them to a log file.
```
Comment (erdemyerebasmaz) [inline src/guide/logging.md:3]:
```suggestion
Breez SDK implements detailed logging via a streaming interface you can manage within your application. The log entries are split into several levels that you can filter and store as desired within your application, for example, by appending them to a log file.
```

**Review by erdemyerebasmaz — COMMENTED** [review]
Looks good. Made suggestions re CI errors.

File: snippets/swift/BreezSDKExamples/Sources/GettingStarted.swift [hunk 5]
```diff
@@ -57,3 +57,15 @@ func gettingStartedNodeInfo(sdk: BlockingBreezServices) {
...
+
+// ANCHOR: logging
+class SDKLogStream: LogStream {
+    func log(l: LogEntry) {
+        print("Received log [", l.level, "]: ", l.line)
+    }
+}
+
+func logging() throws {
+    try? setLogStream(log_stream: SDKLogStream())
```
Comment (erdemyerebasmaz) [inline snippets/swift/BreezSDKExamples/Sources/GettingStarted.swift:69]:
```suggestion
    try? setLogStream(logStream: SDKLogStream())
```

File: snippets/go/getting_started.go [hunk 6]
```diff
@@ -71,3 +71,18 @@ func FetchBalance() {
...
+type SdkLogStream struct{}
+
+func (SdkLogStream) Log(l breez_sdk.LogEntry) {
+	log.Printf("Received log [%v]: %v", l.Level, l.Line)
+}
+
+func GettingStartedLogging() {
+	err := breez_sdk.SetLogStream(SdkLogStream{})
+	if err != nil {
+		log.Fatalf("SetLogStream failed: %#v", err)
```
Comment (erdemyerebasmaz) [inline snippets/go/getting_started.go:85]: Needs to be reformatted according to CI.

File: snippets/csharp/GettingStarted.cs [hunk 7]
```diff
@@ -76,4 +76,27 @@ public void FetchNodeInfo(BlockingBreezServices sdk)
         }
         // ANCHOR_END: fetch-balance
     }
+
+    // ANCHOR: logging
+    public void GettingStartedLogging()
```
Comment (erdemyerebasmaz) [inline snippets/csharp/GettingStarted.cs:81]:
```suggestion
    public void GettingStartedLogging(BlockingBreezServices sdk)
```

File: snippets/go/getting_started.go (hunk 6 above)
Comment (dangeross) [reply snippets/go/getting_started.go:85]: Fixed [ba0b87c](https://github.com/breez/breez-sdk-docs/pull/170/commits/ba0b87ccdf99fa2a1cd2373e6781a1e5d0ce9729)

File: snippets/csharp/GettingStarted.cs (hunk 7 above)
Comment (dangeross) [reply snippets/csharp/GettingStarted.cs:81]: Fixed [ba0b87c](https://github.com/breez/breez-sdk-docs/pull/170/commits/ba0b87ccdf99fa2a1cd2373e6781a1e5d0ce9729), the `sdk = ...` was the issue

Comment (erdemyerebasmaz) [reply snippets/csharp/GettingStarted.cs:81]: Right!

**Review by erdemyerebasmaz — APPROVED** [review]
LGTM

**Pull request merged**